PROJECT

```diff
@@ -1,7 +1,17 @@
 domain: kubewarden.io
+layout:
+- go.kubebuilder.io/v3
+multigroup: true
+projectName: kubewarden-controller-migrated
 repo: github.com/kubewarden/kubewarden-controller
 resources:
-- group: kubewarden
+- api:
+    crdVersion: v1
+    namespaced: true
```

Comment:
Based on https://book.kubebuilder.io/reference/project-config.html, I think this should be false, given that `ClusterAdmissionPolicy` is cluster based.

Reply:
I didn't know kubebuilder introduced that into the `PROJECT` file. I will change it, but in the meantime it looks like the `//+kubebuilder:resource:scope=Cluster` directive inside of `apis/policies/v1alpha1/clusteradmissionpolicy_types.go` works just fine